code/datums/diseases/_disease.dm

```diff
@@ -149,7 +150,7 @@
 	//note that stage is not copied over - the copy starts over at stage 1
 	var/static/list/copy_vars = list("name", "visibility_flags", "disease_flags", "spread_flags", "form", "desc", "agent", "spread_text",
 									"cure_text", "max_stages", "stage_prob", "viable_mobtypes", "cures", "infectivity", "cure_chance",
-									"bypasses_immunity", "spreading_modifier", "severity", "required_organs", "needs_all_cures", "strain_data",
```

Comment:
I removed it from being a list when I added the organ check and forgot to readd it.  Funny that it didn't affect any of my testing since it's a static var that doesn't get changed.

https://github.com/tgstation/tgstation/pull/76766/commits/60401b83284d02d960b611e382710a9ce1f18062